flink-tests/src/test/java/org/apache/flink/test/javaApiOperators/util/CollectionDataSets.java

```diff
@@ -429,7 +418,6 @@ public String toString() {
 	}
 
 	public static class NestedPojo {
-		public static Object ignoreMe;
```

Comment:
I think this one should stay, it is there to check that the type extractor ignores static fields.